core/src/main/java/nl/nn/adapterframework/extensions/javascript/JavascriptEngine.java

```diff
@@ -0,0 +1,21 @@
+package nl.nn.adapterframework.extensions.javascript;
+
+import nl.nn.adapterframework.core.ISender;
+import nl.nn.adapterframework.core.ISenderWithParameters;
+import nl.nn.adapterframework.parameters.ParameterResolutionContext;
+
+public interface JavascriptEngine<E> {
```

Comment:
Try and add more javadoc info to the interface. All implementations inherit the javadoc, which will make it much more readable and future proof!